### PR TITLE
Partner program blog post 1st draft

### PR DESCRIPTION
**Pull request by lindaleebumblebee**
@calcsam this is the first draft of the partner program blog post!

### CODE REVIEW

**Comment by KyleAMathews** [comment]
Deploy preview for *using-postcss-sass* failed.

Built with commit 408b4f7ff4a00634b244c1349398ad2a89d17a44

https://app.netlify.com/sites/using-postcss-sass/deploys/5b60d0aa792f89300da4a7bd

**Comment by KyleAMathews** [comment]
Deploy preview for *gatsbygram* failed.

Built with commit 408b4f7ff4a00634b244c1349398ad2a89d17a44

https://app.netlify.com/sites/gatsbygram/deploys/5b60d0a9792f89300da4a7b7

File: docs/blog/author.yaml [hunk 1]
```diff
@@ -157,3 +157,7 @@
   bio: Software developer and full-time opensourcerer at @contentful. He speaks multiple languages and is often overheard saying "Bonjour" in HTML.
   avatar: avatars/khaled-garbaya.jpg
   twitter: "@khaled_garbaya"
+- id: Linda Watkins
```
Comment (calcsam) [inline docs/blog/author.yaml]: You're already on here a few lines ago from your previous post -- should edit that instead of adding a new bio

**Comment by KyleAMathews** [comment]
Deploy preview for *gatsbyjs* failed.

Built with commit 5df34572828d8de66d19ce8cae116d7089f51500

https://app.netlify.com/sites/gatsbyjs/deploys/5b60d19582d3f17823838bed

File: docs/blog/2018-08-01-partner-program/index.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+	portalId: "4731712",
+	formId: "bbad5044-d31d-4473-bd57-62cb64fdc811",
+	sfdcCampaignId: "701f4000000VDrYAAW"
+});
+</script>
+
+So what’s next for partners?
+
+We’ll have a lot more features and benefits coming soon. Future additions to the program will include:
+
```
Comment (calcsam) [inline docs/blog/2018-08-01-partner-program/index.md]: * Dedicated pages on .org highlighting partners & showcasing the sites they've built with Gatsby

Link if you want it: https://github.com/gatsbyjs/rfcs/blob/8bb84d8a4ed495678ae76645f940a03ad562a7d1/text/0008--individual-agency-company-pages.md#agency-pages

File: docs/blog/2018-08-01-partner-program/index.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+* Lead capture form on your agency’s [Site Showcase](https://next.gatsbyjs.org/showcase/) page so people can get in touch with you directly
+* Live training for developers (in-person or virtual)
+* Co-branded, joint case study highlighted on our blog
```
[the remark this answers is not in the document]
Comment (calcsam) [reply docs/blog/2018-08-01-partner-program/index.md]: "Co-branded" is a bit too marketer-y for this blog's voice

File: docs/blog/2018-08-01-partner-program/index.md [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+tags: ["partner program", "launch", "agency"]
+---
+
+In the business of building websites or web apps for clients? Get the support you need and the visibility you deserve. Gatsby’s Partner Program is here! 
+
+## Grow with Gatsby
+Gatsby is growing like gangbusters due to the contributions of this great open source community—thank you! We’ve been hearing from more and more agencies and website development shops about how they’re embracing Gatsby to build blazing-fast websites and web apps for their clients. We want to support these efforts and help you get the word out about your amazing work. 
+
+## Gatsby Partner Program
+With that in mind, we’re happy to announce Gatsby’s Partner Program. Partner with Gatsby to get your agency or client websites highlighted on the [Gatsby Site Showcase](https://next.gatsbyjs.org/showcase/) to generate more visibility to your work. We’ll also be building out the partner program in the coming months to add more benefits like training for your developers, co-branded case studies, free swag, and more. To show interest in the program, here are a few simple steps to get started.
```
Comment (calcsam) [inline docs/blog/2018-08-01-partner-program/index.md]: "joint" > "co-branded"

**Comment by KyleAMathews** [comment]
Deploy preview for *gatsbyjs* failed.

Built with commit 4ea530544aa63fc7428e752f738dcb2371534ab8

https://app.netlify.com/sites/gatsbyjs/deploys/5b60e47ec6aed678996c7e61

**Pull request merged**